chainercv/links/model/__init__.py

```diff
@@ -0,0 +1 @@
+from chainercv.links.model.segnet.segnet_basic import SegNetBasic  # NOQA
```

Comment:
Can you move imports to `chainercv.links.__init__`?
Also, losses should be moved there as well.